data/jakarta/utils.sh

```diff
@@ -126,3 +126,97 @@ snap_remove()
     snap remove edgexfoundry || true
 }
 
+# wait for services to come online
+# NOTE: this may have to be significantly increased on arm64 or low RAM platforms
+# to accomodate time for everything to come online
+snap_wait_all_services_online()
+{
+    regex="4[0-9][0-9]|5[0-9][0-9]"
+    all_services_online=false
+    i=0
+
+    while [ "$all_services_online" = "false" ];
+    do
+        #max retry avoids forever waiting
+        ((i=i+1))
+        if [ "$i" -ge 300 ]; then
+            echo "services timed out, reached max retry count of 300"
+            exit 1
+        fi
+
+        #dial services
+        core_data_status_code=$(curl --insecure --silent --include \
+            --connect-timeout 2 --max-time 300 \
+            --output /dev/null --write-out "%{http_code}" \
+            -X GET 'http://localhost:59880/api/v2/ping' \
+            -H 'accept: application/json') || true
```

Comment:
passing this header seems unnecessary since we don't care about the response body.

Reply:
Thanks! Removed.